frontend/src/pages/ProgressionChecker/CoursesModal/styles.ts

```diff
@@ -11,14 +11,12 @@ const ModalHeader = styled.div`
 
 const ModalTitle = styled(Title)`
   margin: 0 !important;
-  color: ${({ theme }) => theme.text} !important;
```

Comment:
why are we removing this css for dark mode (same in line ~21) 🥺

Reply:
I ACCIDENTALLY REMOVED ALL THE THEMEMMSEGYAUGBDHSATGJDvhASd

Reply:
wait it appears on my local wtf

Reply:
ohhh i know why know